src/graphics/Window.zig

```diff
@@ -674,8 +682,20 @@ pub fn init() void { // MARK: init()
 
 	// NOTE(blackedout): Since the Vulkan loader is linked statically for Cubyz on macOS, libvulkan*.dylib is part of the Cubyz executable
 	// and GLFW's default attempt to load it dynamically would fail. Instead, tell GLFW where it can find the loader functions directly.
+	//
+	// Additionally, before GLFW touches Vulkan, tell the Vulkan loader where it can find the directory of the layer manifest files and
+	// the MoltenVK driver manifest file by setting the environment variables `VK_ADD_LAYER_PATH` and `VK_DRIVER_FILES`.
+	// Documented at https://vulkan.lunarg.com/doc/view/latest/mac/layer_configuration.html (2025-11-02)
+	// and at https://vulkan.lunarg.com/doc/view/latest/mac/LoaderDriverInterface.html (2025-11-02)
```

Comment:
I think this comment is starting to get too long. maybe it doesn't need two links and it shouldn't repeat information already present in the code "by setting the environment variables `VK_ADD_LAYER_PATH` and `VK_DRIVER_FILES`"